python/scannerpy/database.py

```diff
@@ -244,14 +269,87 @@ def start_cluster(self, master, workers):
             master: ssh-able address of the master node.
             workers: list of ssh-able addresses of the worker nodes.
         """
-        master_cmd = 'python -c "from scannerpy import Database as Db; Db().start_master()"'
-        worker_cmd = 'python -c "from scannerpy import Database as Db; Db().start_worker()"'
-
-        master = self._run_remote_cmd(master, master_cmd)
-        workers = [self._run_remote_cmd(w, worker_cmd) for w in workers]
-        master.wait()
-        for worker in workers:
-            worker.wait()
+
```

Comment:
Why are `start_worker` and `start_master` exposed as top-level public functions, but `start_cluster